src/t8_cmesh.h

```diff
@@ -769,6 +769,17 @@ void                t8_cmesh_new_translate_vertices_to_attributes (t8_locidx_t
                                                                    *attr_vertices,
                                                                    int
                                                                    num_vertices);
+
+/**
+ * \warning This function is only available in debug-modus and should only 
+ * be used in debug-modus.
+ * 
+ * Prints the vertices of each tree of each process
+ * 
+ * \param[in] cmesh   Source-cmesh, which trees get printed.
+ */
+void                t8_cmesh_debug_print_trees (const t8_cmesh_t cmesh,
```

Comment:
Put this function also in debug clause. This will error in case of `-Werror=unused-function`.

Reply:
Just happened in the CI^^